react/week3/ToDoList.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+
+
```

Comment:
I think you should more focus on breaking into sub components, putting all functionality wont give you benefit of react app.